### ISSUE TITLE
Add ability to log in with GitHub credentials

### ISSUE THREAD

**Issue by gvwilson** [swcarpentry/amy#814]
Following up on [this thread](http://lists.software-carpentry.org/pipermail/amy-dev/2016-May/000007.html), add Python Social Auth so that people can log in with GitHub credentials. This will make life easier for folks who are mentoring/going through instructor training.

**Comment by chrismedrela** [comment]
@pbanaszkiewicz, I have some troubles installing `python-social-auth`. What is the result of executing:
```
pip3 install python-social-auth
python3 -c "import social"
```
on your machine?

**Comment by pbanaszkiewicz** [comment]
@chrismedrela it just works, no output.

**Comment by pbanaszkiewicz** [comment]
@narayanaditya95 can you please test what https://github.com/swcarpentry/amy/issues/814#issuecomment-223604528 says at your machine?

**Comment by narayanaditya95** [comment]
It prints:

```
$ python -c "import social; print(social.__version__)"
0.2.19
```

**Comment by chrismedrela** [comment]
OK, I figured it out: `dist-packages` directory was missing in `sys.path`, so, as a workaround, I added it to my PYTHONPATH.

**Comment by pbanaszkiewicz** [comment]
Closed via #825.
